control_panel_api/permissions.py

```diff
@@ -19,6 +19,9 @@ class IsSuperuser(BasePermission):
     def has_permission(self, request, view):
         return is_superuser(request.user)
 
+    def has_object_permission(self, request, view, obj):
```

Comment:
Why to implement this method if it will return always the same result as the `has_permission` method? What does it gives us now?

It's a genuine question because as far as I remember by reading the permissions documentation `has_permission()` is always called first anyway on member endpoints, and `has_object_permission()` is only called for member endpoints if `has_permission()` returned true.

Basically unless I'm missing something or I don't remember correctly if  `has_permissions()` returns `False` (or raise) the `has_object_permission()` is never called.

Worth re-checking the documentation tho.

Reply:
Yep, this is unnecessary